src/analysis/multiSpecies/SteadyCom/subroutines/setCplexParam.m

```diff
@@ -0,0 +1,114 @@
+function LP = setCplexParam(LP, solverParams,verbFlag)
```

Comment:
A similar function already exists in `src/base/solvers`. May we move this file there and eventually later combine both?